template/create-broadcast-group.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.2/dist/css/bootstrap.min.css"
+      rel="stylesheet"
+      integrity="sha384-Zenh87qX5JnK2Jl0vWa8Ck2rdkQ2Bzep5IDxbcnCeuOxjzrPF/et3URy9Bv1WTRi"
+      crossorigin="anonymous"
+    />
+    <title>PostIt - Create Broadcast Group</title>
+  </head>
+  <body>
+    <nav class="navbar navbar-dark bg-dark">
+      <h2 class="navbar-text">PostIt - Create a Broadcast Group</h2>
+    </nav>
+    <section class="vh-100" style="background-color: #508bfc">
+      <div class="container py-5 h-100">
+        <div class="row d-flex justify-content-center align-items-center h-100">
+          <div class="col-12 col-md-8 col-lg-6 col-xl-5">
+            <div class="card shadow-2-strong" style="border-radius: 1rem">
+              <div class="card-body p-5 text-left">
+
+
+                <div class="mb-3 d-grid gap-3">
+                    <label for="broadcastGroup" class="form-label"
+                      >Create broadcast group</label
+                    >
+                    <input
+                      type="input"
+                      name="groupName"
+                      placeholder="Enter group name"
+                      class="form-control"
+                      id="broadcastGroup"
+                    />
+
+                    <button type="submit" class="btn btn-primary col-12 p-2">
+                      Create Group
+                    </button>
+                
+                </div>
+              
+
+                <hr class="my-4" />
+                <p><a href="add_users_group.html">Add users to group</a></p>
```

Comment:
Update this link to link to the new renamed page. Do the same for any other links.
```suggestion
                <p><a href="add-users-group.html">Add users to group</a></p>
```